cmd/common_test.go

```diff
@@ -165,3 +167,131 @@ func TestIsMaster(t *testing.T) {
 		assert.Equal(t, tc.isMaster, isMaster(), tc.name)
 	}
 }
+
+func TestMapToCISVersion(t *testing.T) {
+
+	viperWithData, err := loadConfigForTest()
+	if err != nil {
+		t.Fatalf("Unable to load config file %v", err)
+	}
+	kubeToCISMap, err := loadVersionMapping(viperWithData)
+	if err != nil {
+		t.Fatalf("Unable to load config file %v", err)
+	}
+
+	cases := []struct {
+		kubeVersion string
+		succeed     bool
+		exp         string
+	}{
+		{kubeVersion: "1.11", succeed: true, exp: "cis-1.3"},
+		{kubeVersion: "1.12", succeed: true, exp: "cis-1.3"},
+		{kubeVersion: "1.13", succeed: true, exp: "cis-1.4"},
+		{kubeVersion: "1.16", succeed: true, exp: "cis-1.4"},
+		{kubeVersion: "unknown", succeed: false, exp: ""},
```

Comment:
Maybe add a test for a version lower than we support e.g. 1.10, which should fail